### ISSUE TITLE
Optimize the CNN

### ISSUE THREAD

**Issue by GoatedChopin** [GoatedChopin/XRayBot#1]
Try adding dropout layers after Maxpooling layer with 0.2 dropout.
Try training a model on just the NoFindings and one other class.

**Comment by GoatedChopin** [comment]
https://stackoverflow.com/questions/57787386/how-to-increase-accuracy-in-convolutional-neural-network#:~:text=After%20each%20convolutional%20layer%2C%20apply%20BatchNormazalization%20After%20each,reducing%20the%20batch%20size%2C%20maybe%2032%20or%2016

**Comment by GoatedChopin** [comment]
https://keras.io/keras_tuner/
https://www.youtube.com/watch?v=vvC15l4CY1Q

Convert the model fit function to .fit() from .fit_generator(). The method .fit() supports generators, so this should be a simple syntax change.
